dts/arm/nxp/nxp_rt10xx.dtsi

```diff
@@ -230,9 +230,13 @@
 		};
 
 		ccm: ccm@400fc000 {
-			compatible = "nxp,imx-ccm";
+			compatible = "nxp,imx-ccm", "nxp,imx-rt10xx-ccm";
```

Comment:
`nxp,imx-ccm` is included within `nxp,imx-rt10xx-ccm.yaml` file so is it not enough to keep `nxp,imx-rt10xx-ccm` only?

<pre>
compatible = "nxp,imx-rt10xx-ccm";
</pre>

Reply:
`clock_control_mcux_ccm.c` driver relies on `nxp,imx-ccm`, which is shared across rt10xx and imx8mm families. Here we introduce `nxp,imx-rt10xx-ccm` bindings, which are "extension" just for rt10xx. Those addiitonal properties are used only by rt10xx SoC specific initialization code (at least for now), while all the other are generic enough for `clock_control_mcux_ccm.c` to work as it was before.

Reply:
It's perfectly clear now, thanks.